compiler/noirc_evaluator/src/ssa/ir/dfg.rs

```diff
@@ -308,7 +309,10 @@ impl DataFlowGraph {
         existing_id: Option<InstructionId>,
     ) -> InsertInstructionResult {
         if !self.is_handled_by_runtime(&instruction) {
-            panic!("Attempted to insert instruction not handled by runtime: {instruction:?}");
+            // BUG: With panicking it fails to build the `token_contract`; see:
+            // https://github.com/AztecProtocol/aztec-packages/pull/11294#issuecomment-2624379102
+            // panic!("Attempted to insert instruction not handled by runtime: {instruction:?}");
```

Comment:
https://github.com/AztecProtocol/aztec-packages/pull/11294/commits/c30734b6848daca0f664d11a99a3a158eaf4d700

Reply:
Note that this only occurs with `--inliner-aggressiveness 0`

Reply:
I've added a temporary check to our CI to compile the noir contracts with this setting.